config/config.kinesis.reference.hocon

```diff
@@ -125,6 +125,13 @@
     "iglu:com.acme/skipped3/jsonschema/1-*-*",
     "iglu:com.acme/skipped4/jsonschema/*-*-*"
   ]
+
+  # -- Schemas for which to use the legacy column style used by the v1 BigQuery loader
+  # -- For these columns, there is a column per _minor_ version of each schema.
+  "legacyColumns": [
+    "iglu:com.acme/legacy/jsonschema/1-*-*"
+    "iglu:com.acme/legacy/jsonschema/2-*-*"
```

Comment:
With first version we had a boolean toggle where we could just say 'I don't want v2 for now, just give me v1 for all columns'. Now If I want that behaviour, I have to list all my schemas here (well at least all names with `*-*-*` version pattern)?

I wonder if such toggle to disable v2 entirely could still be useful from user point of view. Maybe migration would be easier if we do the opposite - make `v2Schemas` configurable and all legacy by default...? Or some combination of boolean toggle + legacy schema list config.

Reply:
@stanch please could you comment on this idea?

I implemented it the current way based on a conversation we had.  So users are forced to start using the new column style if they track a new entity.  Based on that plan, I think there is no need to add a flag which turns on v1-by-default.  And I liked that plan because it keeps the implementation simpler.

But now is the time to comment if we do want to allow v1-by-default for a subset of users.

Reply:
Actually @pondzix I don't think we need to wait for Nick's comment.  I am confident this is implemented exactly as we discussed, and as written about in some internal document.

If closer to release we decide to implement v1-by-default as an option, then we can do so as an extra feature request.